handlers/Home/components/Services/styles.css

```diff
@@ -0,0 +1,135 @@
+/** @define HomeServices; use strict */
+
+.HomeServices-list {
+  border-top: 1px solid white;
+  margin: 0;
+  padding: 0;
+}
+
+.HomeServices-list:after {
+  content:'';
+  display:block;
+  clear: both;
+}
+
+.HomeServices-item {
+  border-bottom: 1px solid white;
+  border-right: 1px solid white;
+  box-sizing: border-box;
+  display: inline-block;
+  float: left;
+  height: 25vw;
+  margin-bottom: 0px;
+  position: relative;
+  width: 33.3333%;
+  background-size: cover;
+}
+
+.HomeServices-item:after, .HomeServices-item:before {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+
+}
+
+.HomeServices-item:nth-child(3), .HomeServices-item:nth-child(6) {
+  border-right: 0;
+}
+
+.HomeServices-link {
+  text-decoration: none;
+  color: #393939;
+}
+
+.HomeServices-overlay {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+  z-index: 2;
+  background-color: rgba(250, 171, 24, 0.60);
+}
+
+.HomeServices-content {
+  background-color: transparent;
+  padding: 0 8%;
+  height: 100%;
+  position: relative;
+  z-index: 3;
+}
+
+.HomeServices-title {
+  font-size: 1.6em;
+  font-weight: 300;
+  padding-top: 36%;
+}
+
+.HomeServices-description {
+  display: block;
+  opacity: 0;
+  font-size: 1.1em;
+  font-weight: 300;
+  line-height: 1.4em;
+  transform: translate3d(0, 0, 0);
+}
+
+.HomeServices-item.is-active .HomeServices-content {
+  background-color: rgba(255,255,255,0.92);
+  box-shadow: 0px 2px 15px 0px rgba(0,0,0,0.23);
+  transition: all ease-out 350ms;
+}
+
+.HomeServices-item.is-active  .HomeServices-title {
+  /*padding-top: 12%;*/
+  color: #FAAB18;
+  transition: all ease-out 150ms;
+  transition-delay: 100ms;
+  transform: translate3d(0, -48%, 0);
+
+}
+
+.HomeServices-item.is-active  .HomeServices-description {
```

Comment:
Interestingly, it doesn't break it. This is changes a bit in the second PR though ```.is-active``` changes to ```:hover```, essentially what's happening is parent is being hovered and each nested div is being styled slightly different